src/main/java/com/miraisolutions/xlconnect/Workbook.java

```diff
@@ -997,6 +1023,25 @@ Name getName(String name) {
             throw new IllegalArgumentException("Name '" + name + "' does not exist!");
     }
 
+    List<Name> getNames(String name) {
+        return Collections.unmodifiableList(workbook.getNames(name));
+    }
+
+    private Name getNameInWorksheetScope(String worksheetScope, String name) {
```

Comment:
Not sure it is convenient to have `getNameInWorksheetScope()` as opposed to call it still `getName()`.

By just calling this `getName()` it is just more natural to see that pretty much all calls to the single-argument `getName()` are changed to this two-args version as part of this feature.